installer/mac/ChainCore/Config.swift

```diff
@@ -7,30 +7,27 @@ class Config: NSObject {
 
     /// Checks if the port is in use by another process.
     static func portInUse(_ port: UInt) -> Bool {
-        let sock = socket(PF_INET, SOCK_STREAM, IPPROTO_TCP)
-        if sock <= 0 {
-            return false
-        }
+        var output : [String] = []
+
+        let task = Process()
+        task.launchPath = "/usr/sbin/lsof"
```

Comment:
Yeah, might make sense in the long run.  But I'm also down to punt on a cored change (and the corresponding point release) in favor of a short-term fix that only concerns the Mac app.